web-bundle/src/main/java/com/graphhopper/resources/PtRouteResource.java

```diff
@@ -203,6 +205,15 @@ private CustomPtLeg getCustomPtLeg(Trip.PtLeg leg) {
             logger.info("Failed to find route info for route " + leg.route_id + " for PT trip leg " + leg.toString());
         }
 
-        return new CustomPtLeg(leg, stableEdgeIdsList, routeInfo[0], routeInfo[1], routeInfo[2], routeInfo[3]);
+        // Add proper GTFS feed ID as prefix to all stop names in Leg
```

Comment:
works for now, but can we not just update the actual stop id to the new format? (i'm sure it's more invasive... just curious)

Reply:
It's doable that way I _think_, but it would require messing with code that GH is already using an external library for (Conveyal's gtfs_lib, to be exact). Probably more trouble than it's worth, unfortunately. 

We could in theory add a step at the end of the GTFS import process that updates all of the stop IDs within GH itself (once it's loaded everything through gtfs_lib), but at that point, it seems almost the same as what we have in the PR here 🤷